client/store/reviews.js

```diff
@@ -22,12 +28,24 @@ export const fetchProductReviews = id => {
   };
 };
 
+export const addProductReview = newReview => {
+  return dispatch => {
+    return axios
+      .post('api/reviews', newReview)
+      .then(res => res.data)
+      .then(review => dispatch(addReview(review)))
+      .catch(err => console.log(err));
```

Comment:
Prof says should not to catch errors here, but instead when dispatched in component.

```suggestion
      .catch(err => throw new Error(err));
```